### PR TITLE
Feature/short name support

### PR DESCRIPTION
**Pull request by MarcinIN**

### CODE REVIEW

**Comment by MarcinIN** [comment]
@matiduck @jmarlew  @xHoody  any insights?

File: src/inausoft.netCLI/OptionAttribute.cs [hunk 1]
```diff
@@ -26,11 +31,28 @@ public class OptionAttribute : Attribute
...
         {
-            Name = name ?? throw new ArgumentNullException(nameof(name));
+            if (naming == null)
+            {
+                throw new ArgumentNullException(nameof(naming));
+            }
+
+            var names = naming.Split('|');
+
+            if(names.Length > 2 || names.Length == 0)
```
Comment (xHoody) [inline src/inausoft.netCLI/OptionAttribute.cs:45]: formating ;)

File: src/inausoft.netCLI/OptionAttribute.cs [hunk 2]
```diff
@@ -26,11 +31,28 @@ public class OptionAttribute : Attribute
...
+            {
+                throw new ArgumentException(nameof(naming));
+            }
+
+            Name = names[0];
+
+            if(names.Length == 2)
```
Comment (xHoody) [inline src/inausoft.netCLI/OptionAttribute.cs:52]: form

File: src/inausoft.netCLI/OptionAttribute.cs [hunk 3]
```diff
@@ -26,11 +31,28 @@ public class OptionAttribute : Attribute
...
+            {
+                ShortName = names[1];
```
Comment (xHoody) [inline src/inausoft.netCLI/OptionAttribute.cs:54]: There is no limit for ShortName? It can be as long as we want?

Comment (MarcinIN) [reply src/inausoft.netCLI/OptionAttribute.cs:54]: I guess no.. Should there be one?

**Review by xHoody — CHANGES_REQUESTED** [review]
Some things to correct/answer

File: src/inausoft.netCLI/OptionAttribute.cs (hunk 3 above)
Comment (xHoody) [reply src/inausoft.netCLI/OptionAttribute.cs:54]: maybe not, leave as it is :)

**Pull request merged**